### PR TITLE
no, it still emits death event

### PR DESCRIPTION
**Pull request by NotDemonix**

### CODE REVIEW

File: src/Core/Cluster.js [hunk 1]
```diff
@@ -269,13 +269,16 @@ class Cluster extends EventEmitter {
...
-    _handleExit(respawn = this.manager.respawn, options = {}) {
+    _handleExit(respawn = this.manager.respawn, options = {reason: null}) {
         /**
          * Emitted upon the cluster's child process/worker exiting.
          * @event Cluster#death
          * @param {Child|Worker} process Child process/worker that exited
-         */
-        this.emit('death', this.thread.process);
+         */   
+        if (options.reason != 'Reclustering') {
```
Comment (meister03) [inline src/Core/Cluster.js:278]:
```js
if (options.reason != 'Reclustering') {
```
Change to
```js
if (!options.reason) {
```

**Review by meister03 — CHANGES_REQUESTED** [review]

File: src/Core/Cluster.js (hunk 1 above)
Comment (NotDemonix) [reply src/Core/Cluster.js:278]: then lets just remove reason and put `emitdeath` boolean

**Review by NotDemonix — COMMENTED** [review]
changed to emitdeath boolean

**Comment by NotDemonix** [comment]
wonder when meister review it again cuz i changed some stuff after his change request..

File: src/Plugins/ReCluster.js [hunk 2]
```diff
@@ -131,7 +131,7 @@ class ReClusterManager {
         if (oldClusters.size) {
             this.manager._debug('[↻][ReClustering] Killing old clusters');
             for (let [id, cluster] of oldClusters) {
-                cluster.kill({ force: true, reason: 'Reclusterig' });
+                cluster.kill({ force: true, , emitdeath: false });
```
Comment (TsWin) [inline src/Plugins/ReCluster.js:134]: Double comma please remove one

**Review by TsWin — CHANGES_REQUESTED** [review]
- Fix your double comma
- Maybe also change `emitdeath` variable into camelcase format `emitdeath` => `emitDeath`

**Review by NotDemonix — COMMENTED** [review]
Done!

**Review by TsWin — APPROVED** [review]
Code consistency is ok for me, I'll just wait @meister03 to check if modifications that you did are the changes that he wants.

**Comment by NotDemonix** [comment]
> Code consistency is ok for me, I'll just wait @meister03 to check if modifications that you did are the changes that he wants.

he told me yesterday that he will check it today

**Comment by TsWin** [comment]
> > Code consistency is ok for me, I'll just wait @meister03 to check if modifications that you did are the changes that he wants.
> 
> he told me yesterday that he will check it today

Well, you just need to be patient 😉

**Comment by NotDemonix** [comment]
its meister decision to use this or close it as 1.7.3 fix death event still emmiting, but uses reason instead of emitDeath boolean

**Pull request closed without merging**